ComfyUI/custom_nodes/luma_photon_node/luma_photon_node.py

```
@@ -0,0 +1,238 @@
+            node_id=unique_id,
+            auth_kwargs=auth_kwargs,
+        )
+        response_poll = operation.execute()
+
+        try:
+            img_response = requests.get(response_poll.assets.image)
+            img_response.raise_for_status()
+            img = process_image_response(img_response)
+            return (img, depth_map_to_return)
+        except requests.RequestException as e:
+            logger.error(f"Failed to download generated image: {e}")
```

Ruff (0.12.2)

Undefined name `logger`

(F821)